src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -225,25 +223,26 @@ namespace OpenMS
 
     /**
      * @brief encapsulates readSpectra with file loading convenience if only a file path is given
+     *
      * @param spectra_file file to the SpectrumContainer, must be valid path to mz file
      */
-    void readMzFile(const String& spectra_file)
+    void readMzFileMetaData(const String& spectra_file)
```

Comment:
Is this function used anywhere (besides a test)? Functionality is already included in `addMissingSpectrumReferences`, so could the function be removed?